src/components/infoCard/AddressSection.tsx

```diff
@@ -0,0 +1,73 @@
+import _filter from 'lodash/filter';
+import React from 'react';
+import { StyleSheet, TouchableOpacity } from 'react-native';
+
+import { colors, normalize } from '../../config';
+
+import { formatAddress, locationLink, locationString, openLink } from '../../helpers';
+import { location } from '../../icons';
+import { Address } from '../../types';
+import { Icon } from '../Icon';
+import { RegularText } from '../Text';
+import { InfoBox } from '../Wrapper';
+
+type Props = {
+  address?: Address;
+  addresses?: Address[];
+};
+
+const addressOnPress = (address?: string) => {
+  const mapsString = locationString(address);
+  const mapsLink = locationLink(mapsString);
+
+  openLink(mapsLink);
+};
+
+export const AddressSection = ({ address, addresses }: Props) => {
+  if (!address && !addresses?.length) {
+    return null;
+  }
+
+  const filteredAddresses = _filter(addresses ?? [], (address) => address.kind === 'default');
+
+  // we do not check for address.kind here as we passed in a specific address, that we want to show
+  if (address) filteredAddresses.unshift(address);
+
+  return (
+    <>
+      {filteredAddresses.map((item, index) => {
+        const address = formatAddress(item);
+
+        const isPressable = item.city?.length || item.street?.length || item.zip?.length;
```

Comment:
is it enough to have one of those? or do we want the address click just in case of a complete address?

Reply:
i think it is reasonable to open the map then, because the map app will then search for a location based on the given information.
do you think we should be more strict here?

Reply:
don't care. lets see, what map apps will be able to do and if these cases will occur.